### ISSUE TITLE
[TSDoc> Search]: Ensure link i.e API Extractor are distinguished from surrounding text in a way that does not rely on color

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/rushstack-websites#143]
**Test Environment:**
OS Version: 22H2 (OS Build 25370.1)
Browser: Edge Dev (Version 115.0.1866.1 (Official build) dev (64-bit))
URL: https://tsdoc.org/
Tool: Accessibility Insights for Web

**Repro Steps:**
1. Open the above URL in edge dev browser and login with v-.
2. Run the fast pass of accessibility insights for web.
3. Observe the issue.

**Actual Result:**
API Extractor link has insufficient color contrast of 2.356:1 with the surrounding text which is less than 3:1.

**Expected Result:**
Ensure link i.e API Extractor are distinguished from surrounding text in a way that does not rely on color

**Snippet:** 
<a href="https://api-extractor.com/" target="_blank" rel="noopener noreferrer">API Extractor</a>

**How to fix:** 
Fix any of the following:
  The link has insufficient color contrast of 2.35:1 with the surrounding text. (Minimum contrast is 3:1, link text: #006721, surrounding text: #1c1e21)
  The link has no styling (such as underline) to distinguish it from the surrounding text

**Notes:**
1. Same issue repro on 'TSDoc Playground' link, isInlineTag text, Suggest an update, Learn more, and How can I use TSDoc link.
2. Same issue repro for 'Trimming based on related tags' link under 'Tags' link in the navigation landmark.
3. Same issue repro on 'Rush Stack' link under 'Help' in the main navigation landamrk.
4. Same issue repro on 'Building the projects', recommended practices, Everyday commands and Microsoft Open Source Code of Conduct link under 'Submitting a pull request' in 'Docs sidebar navigation landmark'.

**User Impact:**
Person with visual disability will get impacted as user will not be able to differentiate the link from the text and hence will not be able to interact with the content easily.

**WCAG Reference Link:**
https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

<img width="960" alt="MAS1 4 1 Ensures links are distinguished from text" src="https://github.com/microsoft/rushstack-websites/assets/104483005/05bccab7-4f9d-429f-982b-9cc25977c0ba">
<img width="960" alt="MAS1 4 1 Luminosity ratio of the text from surrounding is less than required" src="https://github.com/microsoft/rushstack-websites/assets/104483005/34b45c4c-82ab-4f97-89d7-2874f045450e">

**Comment by vaishali1397** [comment]
Closing this issue and will track with bug id: https://github.com/microsoft/rushstack-websites/issues/169
